### PR TITLE
Add Apache Flink release 1.10.2

### PR DESCRIPTION
**Pull request by zhuzhurk**
Adds release announcement and download link for 1.10.2
Dates on announcements are still TBD and should be updated accordingly once release happens.

### CODE REVIEW

**Review by uce — APPROVED** [review]
LGTM mod dates.

File: _posts/2020-08-11-release-1.10.2.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18097'>FLINK-18097</a>] -         History server doesn&#39;t clean all job json files
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18168'>FLINK-18168</a>] -         Error results when use UDAF with Object Array return type
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18223'>FLINK-18223</a>] -         AvroSerializer does not correctly instantiate GenericRecord
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18241'>FLINK-18241</a>] -         Custom OptionsFactory in user code not working when configured via flink-conf.yaml
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18242'>FLINK-18242</a>] -         Custom OptionsFactory settings seem to have no effect on RocksDB
```
Comment (Myasuka) [inline _posts/2020-08-11-release-1.10.2.md]: it does not tell user explicitly that any previous program depending on 'RocksDBOptionsFactory' or extending `DefaultConfigurableOptionsFactory` might be broken as we remove `RocksDBOptionsFactory#createDBOptions(DBOptions)`  and `RocksDBOptionsFactory#createColumnOptions(ColumnFamilyOptions)` interfaces to fix this issue.

**Review by Myasuka — CHANGES_REQUESTED** [review]

File: _posts/2020-08-11-release-1.10.2.md (hunk 1 above)
Comment (zhuzhurk) [reply _posts/2020-08-11-release-1.10.2.md]: A new note section is added to show the possible incompatible changes of FLINK-18242.
Would you take another look?

Comment (Myasuka) [reply _posts/2020-08-11-release-1.10.2.md]: Thanks for your fix, LGTM now.

**Comment by zhuzhurk** [comment]
Thanks for reviewing @uce @Myasuka 
We can announce the release today. I have updated the date to 2020-08-25 and will merge the change soon.

**Pull request merged**